src/index.ts

```diff
@@ -71,40 +158,80 @@ const plugin: JupyterFrontEndPlugin<void> = {
     script.type = 'text/javascript';
     script.async = true;
     script.onload = () => {
-      // Define the widgets base module for RequireJS
-      (window as any).define('@jupyter-widgets/base', [], () => base);
-
-      const commandID =
-        '@jupyterlab/plugin-playground:LoadCurrentFileAsExtension';
-      app.commands.addCommand(commandID, {
+      // Define the widgets base module for RequireJS - left for compatibility only
+      (window as any).define(
+        '@jupyter-widgets/base',
+        [],
+        () => modules['@jupyter-widgets/base']
+      );
```

Comment:
Because imports are now handled in `resolveImportStatement`, we can remove the unsafe `require` loading. To do so without breaking compatibility with existing code I will pass `require` instance as an argument to the executor function.

Reply:
This was implemented in ef7c8a58c4d54f9121be528647010837ee35f33e. This has a number of advantages:
- prevents polluting the global `window` with `require`, `define` and `requirejs`,
- allows to use the plugin playground when offline, and
- ensures that a known version of requirejs is loaded over something which might change once `requirejs.org` expires/gets taken over

It was ridiculously difficult to accomplish using standard methods, so instead it is just using a `raw-loader` + iframe.

In short there is no sane way to load requirejs via webpack: it was written before any module standards were a thing and not updated much recently; the `npm` package has `main` field pointing to `r.js` which is a big blob with everything compiled for node, and there is no `browser` field; even after using `alias` and `rules` the `requirejs` variables get optimized out by webpack and for some strange reason enforcing `sideEffect`s does not help (I don't know, maybe this gets overwritten by JupyterLab config?). `@types/requirejs` are quirky too.